hardhat.config.js

```diff
@@ -25,11 +25,10 @@ const { argv } = require('yargs/yargs')()
       type: 'string',
       default: 'contracts',
     },
-    mode: {
-      alias: 'compileMode',
-      type: 'string',
-      choices: ['production', 'development'],
-      default: 'development',
+    runs: {
+      alias: 'optimizationRuns',
+      type: 'number',
+      default: 200,
```

Comment:
Any reason for picking 200 in particular? Feels pretty arbitrary and I'd rather just do 1.
Ideally we should have tests and benchmarks with both few and a lot of runs

Reply:
I think 200 is the canonical value that "most" project use. I don't know where that value comes from exactly, but it has been around since I started using truffle in 2017. 

For example, its the default value used by remix.

Reply:
That is also the value we used before when running with optimization (for example when doing gas comparaison).

Reply:
it's also the default value of `solc --optimize`: https://docs.soliditylang.org/en/latest/using-the-compiler.html#optimizer-options